modules/reliability/php/NDB_Menu_Filter_reliability.class.inc

```diff
@@ -69,32 +69,37 @@ class NDB_Menu_Filter_reliability extends NDB_Menu_Filter
         foreach($pscRows AS $row) {
             $this->centerIDMap[$row['CenterID']] = $row['Name'];
         }
-        
+
         $this->_populate_reliability_table();
 
         $query = " FROM candidate, session, flag, reliability WHERE reliability.CommentID=flag.CommentID AND candidate.CandID=session.CandID AND session.ID=flag.SessionID";
 
         // only view their own profiles, unless they have permission to see all
         if (!($user->hasPermission('reliability_edit_all') || $user->hasPermission('access_all_profiles'))) {
-            $query .= " AND candidate.centerID = '" . $user->getData('CenterID') . "' "; 
+            $query .= " AND candidate.centerID = '" . $user->getData('CenterID') . "' ";
         }
         $config=&NDB_Config::singleton();
         $useProjects = $config->getSetting("useProjects");
         // set the class variables
-        $this->columns = array('candidate.PSCID', 'reliability.reliability_center_id AS Site', 'session.SubprojectID as Cohort', 'session.CandID AS DCCID', 'candidate.Gender', 'session.Visit_label');
+        $this->columns = array('candidate.PSCID', 'reliability.reliability_center_id', '(SELECT name FROM psc WHERE CenterID=reliability.reliability_center_id) as site', 'session.SubprojectID as Cohort', 'session.CandID AS DCCID', 'candidate.Gender', 'session.Visit_label');
```

Comment:
@driusan  It is changed because in the old version the conversion between numeric CenterID and the actual name was done on the front-end. Now, I get this information in the query right away. 

Don't know how a JOIN would benefit here.